iam_mapping.py

```diff
@@ -46,7 +54,40 @@ def delete_mapping(body: dict, meta: dict, spec: dict, **kwargs):
     apply_mapping(cm, updated_mapping)
 
 
-def deploy_crd_definition():
+@kopf.on.startup()
+def on_startup(logger: logger, **kwargs) -> None:
+    # Do a full synchronization at the start
+    logger.info("Deploy CRD definition")
+    deploy_crd_definition()
+    logger.info("Reconcile all existing ressources")
+    full_synchronize()
+
+
+@kopf.on.probe(id='sync')
+def get_monitoring_status(**kwargs) -> str:
```

Comment:
This function seems a bit useless. It receives kwargs that are not use to call another function that either return a str or throw.

Reply:
The framework expect to have a function with kwargs.